mycart/style.css

```diff
@@ -99,4 +102,13 @@ a {
 
 p.prod-head {
     color: green;
```

Comment:
I would suggest making green a variable because it will be used in many places like discount, quiz correct answer, success toast etc